Manual.md

```diff
@@ -1272,6 +1288,229 @@ cache during the install/removal of the package
+to register/remove entries from the arbitrary executable binary formats database.
+
+To include the trigger use the `binfmts` variable, as the trigger is useless without it.
+
+<a id="triggers_dkms"></a>
+#### dkms
+
+The dkms trigger is responsible for compiling and removing dynamic kernel modules of a
+package.
+
+During installation the trigger compiles and install the dynamic module for all `linux`
+packages that have their corresponding headers installed. During removal the corresponding
+module will be removed
+
+To include the trigger use the `dkms_modules` variable, as the trigger is useless without
+it.
+
+<a id="triggers_gonf_schemas"></a>
+#### gconf-schemas
+
+The gconf-schemas trigger is responsible for registering and removing .schemas and
+.entries files into the schemas database directory
+
+During installation it uses `gconftool-2` to install .schemas and .entries files into
+`usr/share/gconf/schemas`. During removal it uses `gconftool-2` to remove the entries
+and schemas from the database from the package being removed.
+
+To include it add `gconf-schemas` to `triggers` and add the appropriate .schemas in
+the `gconf_schemas` variable and .entries in `gconf_entries`.
+
+The trigger is automatically added if `/usr/share/gconf/schemas` is present as a
+directory in the package. All .schemas files under `/usr/share/gconf/schemas` are
+automatically added to `gconf_schemas`.
+
+<a id="triggers_gio_modules"></a>
+#### gio-modules
+
+The gio-modules trigger is responsible for updating the Glib GIO module cache with
+`gio-querymodules` from the `glib` package
+
+during install and removal it just runs `gio-querymodules` updating the cache file
+present at `usr/lib/gio/modules`
+
+It is automatically included when a package has `/usr/lib/gio/modules` present
+as a directory.
+
+<a id="triggers_gsettings_schemas"></a>
+#### gsettings-schemas
+
+The gsettings-schemas trigger is responsible for compiling Glib's GSettings XML
+schema files during installation and removing the compiled files during removal.
+
+During installation it uses `glib-compile-schemas` from `glib` to compile the
+schemas into files with the suffix .compiled into `/usr/share/glib-2.0/schemas`.
+
+During removal of the glib package it deletes all files inside
+`/usr/share/glib-2.0/schemas` that end with .compiled.
+
+It is automatically included when a package has `/usr/share/glib-2.0/schemas`
+present as a directory.
+
+<a id="triggers_gtk_icon_cache"></a>
+#### gtk-icon-cache
+
+The gtk-icon-cache trigger is responsible for updating the gtk+ icon cache.
+
+During installation it uses `gtk-update-icon-cache` to update the icon cache.
+
+During removal of the gtk+ package it deletes the `icon-theme.cache` file
+in the directories defined by the variable `gtk_iconcache_dirs`
+
+It is automatically added on packages that have `/usr/share/icons` available
+as a directory, all directories under that directory have their absolute path
+added.
```

Comment:
> all directories under that directory have their absolute path added.

Maybe I'm just a little dense right now, but added to what? To `icon-theme.cache`?